au4/src/effects/effects_base/internal/effectexecutionscenario.cpp

```diff
@@ -34,6 +34,13 @@ muse::Ret EffectExecutionScenario::performEffect(const EffectId& effectId)
     return doPerformEffect(project, effectId, 0);
 }
 
+muse::Ret EffectExecutionScenario::repeatEffect(const EffectId& effectId)
+{
+    AudacityProject& project = *reinterpret_cast<::AudacityProject*>(
+        globalContext()->currentProject()->au3ProjectPtr());
```

Comment:
If the project is required several times in a class, it is better to add a helper method to get it, something like this
```
AudacityProject& EffectExecutionScenario::projectRef() const
{
    AudacityProject* project = reinterpret_cast<AudacityProject*>(globalContext()->currentProject()->au3ProjectPtr());
    return *project;
}
```